traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -555,6 +559,35 @@ public HTTPRouteResult multiRoute(final HTTPRequest request, final Track track)
 		return routeResult;
 	}
 
+	public String buildPatternBasedHashString(final DeliveryService deliveryService, final String requestPath) {
+		if (deliveryService.getConsistentHashRegex() != null && !deliveryService.getConsistentHashRegex().isEmpty() && !requestPath.isEmpty()) {
+			return buildPatternBasedHashString(deliveryService.getConsistentHashRegex(), requestPath);
+		} else {
```

Comment:
nit: since the `if` block returns, this `else` block isn't really necessary and could just `return requestPath` outside of the block.